test/mock_query_test.dart

```diff
@@ -175,12 +175,11 @@ void main() {
 
   test('orderBy returns documents sorted by documentID', () async {
     final instance = MockFirestoreInstance();
-    await instance.collection('users').doc('3').set({});
-    await instance.collection('users').doc('2').set({});
-    await instance.collection('users').doc('1').set({});
+    await instance.collection('users').doc('3').set({'value': 3});
```

Comment:
I believe this is due to a bug in MockDocumentReference. I've filed #156 to investigate and fix it later if necessary.